apps/react-18-tests-v9/cypress-react-18.config.ts

```diff
@@ -0,0 +1,30 @@
+import * as path from 'path';
+import { baseConfig } from '@fluentui/scripts-cypress';
+
+// Exclude files that are not compatible with React 18 yet
+const excludedSpecs = [
+  '!' + path.resolve('../../packages/react-components/react-tag-picker/library/src/**/*.cy.{tsx,ts}'),
```

Comment:
with this setup we should probably enable these to make it visible that these are failing. thoughts ?

Reply:
I agree. I can make a follow-up PR, since I'm fixing other tests that are failing.